Material.Styles/Themes/SystemThemeProbe.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Runtime.InteropServices;
+using Avalonia.Logging;
+using Material.Styles.Themes.Base;
+
+namespace Material.Styles.Themes;
+
+public static class SystemThemeProbe {
+    /// <summary>
+    /// Tries to resolve base theme mode for the current system
+    /// </summary>
+    /// <returns>Base theme (<c>Dark</c>/<c>Light</c>) or <c>null</c></returns>
+    public static BaseThemeMode? GetSystemBaseThemeMode() {
```

Comment:
SystemThemeProbe can be implemented for other OS too, not just for Windows I think.
Anyway it is a good attempt to let library choose which theme scheme are suitable by OS user settings.

I think I should note this as a part of 3.0.0 major update to improve this feature

Reply:
I think what we should do everything related to system theme probing in dev branch (and also other features, what can be done independently of Avalonia 11)